mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -228,6 +228,10 @@ The `settings` block supports:
     for more details and supported versions. Postgres supports only shared-core machine types,
     and custom machine types such as `db-custom-2-13312`. See the [Custom Machine Type Documentation](https://cloud.google.com/compute/docs/instances/creating-instance-with-custom-machine-type#create) to learn about specifying custom machine types.
 
+The optional `settings.advanced_machine_features` subblock supports:
```

Comment:
Hmm- this looks different than I expected, but it's self-consistent within the file. Non-blocker, just commenting in case someone is reading this change in the future.

Reply:
https://github.com/hashicorp/terraform-provider-google/issues/14813